cpp/src/io/csv/csv_writer.cu

```diff
@@ -96,189 +114,145 @@ NVStrings* column_to_strings_csv( gdf_column* col, const char* delimiter, const
     return rtn;
 }
 
-//
-// The args structure is interpretted as documented in io_types.h
-// This will create a CSV format by allocating host memory for the
-// entire output and determine pointers for each row/column entry.
-// Each column is converted to an NVStrings instance and then
-// copied into their position in the output memory. This way,
-// one column is processed at a time minimizing device memory usage.
-//
+/**---------------------------------------------------------------------------*
+ * @brief Reads CSV-structured data and returns an array of gdf_columns.
```

Comment:
I think you mean it reads an array of gdf_columns and writes CSV-structured data...